src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,91 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private Object[] elementData = new Object[DEFAULT_CAPACITY];
+    private int size;
+
     @Override
     public void add(T value) {
-
+        resize(size);
+        elementData[size] = value;
+        size += 1;
```

Comment:
```suggestion
        size++;
```
це ще так можна записати